### DISCUSSION TITLE
JWT Question

### DISCUSSION THREAD

**Discussion by danielwagmi** [Web3Auth/Web3Auth#775] [Help]
Rather than using social login, email, or SMS, can we use the JWT provider to accept username and password from the user and allow them to login to other devices using user and password and we authenticate with JWT?

**Comment by rinzler99** [comment]
It is possible, for that you can use auth0 with 'database connection'. Configure a username/password based auth0 tenant i.e. database connection. For reference, please check our existing auth0 [guide](https://web3auth.io/docs/guides/auth0).

**Reply by shahbaz17** [reply]
Also, you could do this with Firebase, AWS Cognito, and any other login providers that providers idToken(JWT) upon user authentication.

**Reply by shahbaz17** [reply]
Hey @danielwagmi 

Did you able to work with the solution provided?

**Comment by danielwagmi** [comment]
Yes it was great sorry for the lack of response !
